### PR TITLE
Changed readme.md

### PR DESCRIPTION
**Pull request by NgLoader**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,78 @@
...
-	<repositories>
+<repositories>
 		<repository>
 		    <id>jitpack.io</id>
 		    <url>https://jitpack.io</url>
 		</repository>
-	</repositories>
-```
-
-Replace `<version>` number with this number: [![](https://jitpack.io/v/TechFortress/GriefPrevention.svg)](https://jitpack.io/#TechFortress/GriefPrevention)
```
Comment (RoboMWM) [inline README.md:46]: Not sure why you'd remove this - it'd be nice to not have to constantly update the readme because of version bumps.

Comment (NgLoader) [reply README.md:46]: Okay

**Comment by NgLoader** [comment]
Like this?

**Comment by RoboMWM** [comment]
info would be better suited in Griefprevention documentation. https://github.com/TechFortress/GriefPrevention-Docs

**Pull request closed without merging**